src/App.js

```diff
@@ -2,15 +2,14 @@ import BitcoinPrice from './components/BitcoinPrice';
 import './App.css';
 import { useState } from 'react';
 
-const UP_TYPE = 'up'
-const DOWN_TYPE = 'down'
+const UP_TYPE = 'up';
```

Comment:
the `;` used to be expected but this is kind of going away
doesn't hurt to do it, but not as needed just fyi